app/uk/gov/hmrc/plasticpackagingtax/registration/views/liability_weight_page.scala.html

```diff
@@ -1,70 +1,88 @@
-@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.sectionHeader
-@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.errorSummary
-@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.saveButtons
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.{errorSummary, paragraphBody, saveButtons, sectionHeader}
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.model.{BackButton, Title}
 
 @this(
-  govukLayout: main_template,
-  paragraphBody: paragraphBody,
-  govukButton: GovukButton,
-  govukInput: GovukInput,
-  sectionHeader: sectionHeader,
-  govukFieldset: govukFieldset,
-  errorSummary: errorSummary,
-  formHelper: formWithCSRF,
-  saveButtons: saveButtons,
+        govukLayout: main_template,
+        paragraphBody: paragraphBody,
+        govukButton: GovukButton,
+        govukInput: GovukInput,
+        sectionHeader: sectionHeader,
+        govukFieldset: govukFieldset,
+        errorSummary: errorSummary,
+        formHelper: formWithCSRF,
+        saveButtons: saveButtons,
+        govukInsetText: GovukInsetText,
 )
 @(form: Form[LiabilityWeight])(implicit request: Request[_], messages: Messages)
 
-@totalWeightField = @{form("totalKg")}
+@totalWeightField = @{
+    form("totalKg")
+}
 
 @errorMessages = @{
     val errors = (form.errors).map(err => messages(err.message, err.key)).mkString("<br>")
     if(errors.nonEmpty) Some(ErrorMessage(content = HtmlContent(errors))) else None
 }
 
+@pptWeightQuestion = {
+    <h1 class="govuk-heading-l">@messages("liabilityWeightPage.question")</h1>
+}
+
+@pptGuidanceLink = {
+    <a id="guidance-link" href="@messages("liabilityWeightPage.guidance.href")" target="_blank">@messages("liabilityWeightPage.guidance.description")</a>
+}
+
+@pptWeightHelp = {
+    <p class="govuk-body">@messages("liabilityWeightPage.info")</p>
+    <p class="govuk-body">@Html(messages("liabilityWeightPage.info2", pptGuidanceLink))</p>
+}
+
 @govukLayout(title = Title("liabilityWeightPage.title")) {
-  @errorSummary(form.errors)
 
-  @sectionHeader(messages("liabilityWeightPage.sectionHeader"))
+    @errorSummary(form.errors)
+
+    @sectionHeader(messages("liabilityWeightPage.sectionHeader"))
 
-  @formHelper(action = pptRoutes.LiabilityWeightController.submit(), 'autoComplete -> "off") {
-    @govukInput(
-    Input(
-        id = s"${totalWeightField.name}",
-        name = totalWeightField.name,
-        value = totalWeightField.value,
-        hint = Some(Hint(content = HtmlContent(Html( messages("liabilityWeightPage.hint"))))),
-        label = Label(
-            isPageHeading = true,
-            classes = s"govuk-label--l",
-            content = Text(messages("liabilityWeightPage.question"))),
-            errorMessage = errorMessages)
+    @pptWeightQuestion
+
+    @pptWeightHelp
+
+    @formHelper(action = pptRoutes.LiabilityWeightController.submit(), 'autoComplete -> "off") {
```

Comment:
The key question here is how successful are screen readers on this page?
@tmc-ee can you check this once this ticket goes in?

Happy to approve it for the time being as I believe is a great compromise.

Reply:
@RyanKellyHMRC any chance we could do this locally on your machine before merging? Doing it up front saves potential faffing afterwards.

Reply:
Paired with @RyanKellyHMRC to check this, the page was read out as expected when using VoiceOver for macOS.